seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-feishu/src/main/scala/org/apache/seatunnel/spark/feishu/FeishuClient.scala

```diff
@@ -16,25 +16,27 @@
  */
 package org.apache.seatunnel.spark.feishu
 
-import scala.collection.mutable.ArrayBuffer
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.{ArrayNode, ObjectNode}
 
-import com.alibaba.fastjson.{JSON, JSONArray, JSONObject}
+import scala.collection.mutable.ArrayBuffer
 import org.apache.http.{HttpEntity, HttpHeaders}
 import org.apache.http.client.methods.{CloseableHttpResponse, RequestBuilder}
 import org.apache.http.impl.client.{CloseableHttpClient, HttpClients}
 import org.apache.http.util.EntityUtils
+import org.apache.seatunnel.common.utils.JsonUtils
 import org.apache.spark.sql.Row
 import org.apache.spark.sql.types.{DataTypes, StructField, StructType}
 import org.slf4j.{Logger, LoggerFactory}
 
 class FeishuClient(appId: String, appSecret: String) {
-  val logger: Logger = LoggerFactory.getLogger(this.getClass)
+  /*val logger: Logger = LoggerFactory.getLogger(this.getClass)
 
   def getToken: String = {
     val url = Config.TOKEN_URL.format(appId, appSecret)
     val result = this.requestFeishuApi(url, null)
     logger.info(s"Request token and get result $result")
```

Comment:
I don't familiarize Scala, anyone could update this

Reply:
I finish the scala code.